pkg/reconciler/kubernetes/kubeclient/kubeclient.go

```diff
@@ -30,12 +29,25 @@ type KubeClient struct {
 	mapper        *restmapper.DeferredDiscoveryRESTMapper
 }
 
+func NewInCluster() (*KubeClient, error) {
+	config, err := rest.InClusterConfig()
```

Comment:
The name is a big tricky to understand without looking into the code, can we make it a bit more self-explaining (e.g. `NewInClusterConfig` or similar)?

Reply:
I have changed it to `NewInClusterClient`